app/models/friendship.rb

```diff
@@ -0,0 +1,18 @@
+class Friendship < ApplicationRecord
```

Comment:
I tried to do the ff:

Created 2 users -> `user1` and `user2`.. Then I created the following friendship:

`user1.friendships.create(friend_id: user2.id)`

So I would expect now `user1.friends` to include `user2` right? And so I ran:
`user1.friends` -> And yes `user2` was there.

But then I also would expect that running `user2.friends` I would see `user1` right?

But that's not the case:
`user2.friends` results in an empty array

Reply:
![Screen Shot 2019-05-14 at 11 32 50 AM](https://user-images.githubusercontent.com/21004010/57703006-16e74b80-7657-11e9-83c2-3a15f963b3d6.png)
I and my partner tried testing friendships on our console and it works fine.
1. I created users `u1` and `u2`
2. I created the friendship `u1.friendships.create(friend_id: u2.id)`
3. I checked for u1 friends by running `u1.friends` and I got an array with `u2` in it
4. I also ran `u2.friends` and I got an array with `u1` in it